MIP106/MIP106.md

```diff
@@ -565,8 +495,9 @@ Spark has a special token pre-farming program based on usage of its lending plat
 The pre-farming period is active from Ethereum block 17956537 [AUG 20 2023 14:25 UTC] and ends at the earlier of two events:
 
 - Start of SubDAO token farming OR MAY 20 2024 14:25 UTC.
-- 30m SPK tokens allocated if full 9 months pass. Pro-rated if earlier.
-- SPK portion sourced from SubDAO workforce bonus pool, available within 1 month after SubDAO farming starts.
+- 30m SPK tokens allocated for the first 9 months.
+- Following the end of the initial 9 month period, and additional pre-farming period will continue until the launch of Spark. Rewards are calculated on a per-block basis according to the Anti-cheat formula, at a rate of 3.33m SPK tokens per month.
```

Comment:
"and" could be changed to "an" before the phrase "additional pre-farming period."